server/src/test/java/org/elasticsearch/index/query/SearchExecutionContextTests.java

```diff
@@ -380,27 +381,28 @@ public void testSearchRequestRuntimeFieldsAndMultifieldDetection() {
     }
 
     public void testSyntheticSourceScriptLoading() throws IOException {
-
         // Build a mapping using synthetic source
         SourceFieldMapper sourceMapper = new SourceFieldMapper.Builder(null).setSynthetic().build();
-        RootObjectMapper root = new RootObjectMapper.Builder("_doc", Explicit.IMPLICIT_TRUE).build(MapperBuilderContext.root(true, false));
+        RootObjectMapper root = new RootObjectMapper.Builder("_doc", Explicit.IMPLICIT_TRUE).add(
+            new KeywordFieldMapper.Builder("cat", IndexVersion.current()).ignoreAbove(100)
+        )
+            .addRuntimeField(new LeafRuntimeField("runtime_field", new KeywordFieldMapper.KeywordFieldType("runtime_field"), List.of()))
```

Comment:
Let's remove the runtime field here, given that it just adds noise?

Reply:
Done.